migrations/general_metricfilter/testcases/merge_fieldexclude/expected.conf

```diff
@@ -0,0 +1,3 @@
+[[inputs.dummy]]
+servers = ["tcp://127.0.0.1:1883"]
+fieldexclude = ["valueA", "valueB", "valueC"]
```

Comment:
Not that it really matters but current telegraf config parser would process them as `["valueB", "valueC", "valueA"]`.